development/prow-installer/pkg/serviceaccount/serviceaccount.go

```diff
@@ -36,19 +32,33 @@ func NewClient(prefix string, iamservice IAM) *Client {
 }
 
 // Creates GKE Service Account. SA name is trimed to 30 characters per GCP limits.
-// If AccessManager has non zero value of prefix field, created ServiceAccounts are prefixed.
-func (client *Client) CreateSA(options SAOptions) (*iam.ServiceAccount, error) {
+func (client *Client) CreateSA(name string, project string) (*iam.ServiceAccount, error) {
 	if client.prefix != "" {
-		options.Name = fmt.Sprintf("%s-%s", client.prefix, options.Name)
+		name = fmt.Sprintf("%s-%s", client.prefix, name)
+	}
+	name = fmt.Sprintf("%.30s", name)
+	project = fmt.Sprintf("projects/%s", project)
+	request := &iam.CreateServiceAccountRequest{
+		AccountId: name,
 	}
-	options.Name = fmt.Sprintf("%.30s", options.Name)
-	options.Project = fmt.Sprintf("projects/%s", options.Project)
-	sa, err := client.iamservice.CreateSA(options.Name, options.Project)
-	if err != nil && !googleapi.IsNotModified(err) {
-		log.Printf("Error: {%v} when creating %s service account in %s project.", err, options.Name, options.Project)
-		return &iam.ServiceAccount{}, err
+	sa, err := client.iamservice.CreateSA(request, project)
+	if err != nil {
+		log.Printf("When creating %s serviceaccount in %s project got error: %w.", name, project, err)
+		return nil, err
 	} else {
-		log.Printf("Created service account:\n %s", options.Name)
+		log.Printf("Created serviceaccount: %s", name)
 		return sa, nil
 	}
 }
+
+// safqdn should be serviceaccount mail. Pass here iam.ServiceAccount.Email returned by Client.CreateSA().
```

Comment:
Can you revisit the comments on top of exported functions? Format should be:
```
// MyFuncName is doing this. It uses `hello` string parameter for modifying this and returns error if it fails.
func MyFuncName(hello string) error {
//....
}
```
The important part is, that the function name is the first part of the comment. Otherwise the linter will complain. It seems our linter setup is not complaining about this at the moment.